### PR TITLE
first changes to support Ubuntu 20.04

### PR DESCRIPTION
**Pull request by vernondcole**
### What does this PR do?

Loads a git version of salt minion onto a pre-release Ubuntu 20.04 machine

### What issues does this PR fix or reference?

#1425

### CODE REVIEW

File: bootstrap-salt.sh [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (vutny) [inline bootstrap-salt.sh]: I guess the blank line here was added occasionally, right?

Comment (vernondcole) [reply bootstrap-salt.sh]: Yes, I suppose so. At least, I did not intend to add it.

Comment (vutny) [inline bootstrap-salt.sh]:
```suggestion
```
Better to remove it, cause it breaks lint checker.

**Review by vutny — COMMENTED** [review]
The PR looks good, @vernondcole . But unfortunately, after merging #1424 it has conflicts now.
Would you mind to update your changes in line with current `develop`?

**Comment by vernondcole** [comment]
I committed these and pushed them straight from my 20.04 test machine, which does not have gpg keys registered.  
@vutny Do I need to re-submit from a registered workstation?

**Review by s0undt3ch — APPROVED** [review]

**Comment by s0undt3ch** [comment]
It's desirable but I can override the key issue.

**Pull request merged**